benchmark/benchmark.py

```diff
@@ -84,102 +79,147 @@ def parse_numpy_benchmark(filename):
     return "".join(lines)
 
 
-def get_numpy_benchmarks():
-    root = Path(__file__).resolve().parent / "benchmarks"
+def get_benchmark_scripts(scripts_dir="benchmarks/numpy", repeat=11, number=5):
```

Comment:
Do we want the default argument for `scripts_dir`? Not that it makes any difference...

Reply:
No, I think it doesn't need the default argument, removed.